### PR TITLE
Expand documentation on SymbolicFunction

### PR DESCRIPTION
**Pull request by dbarbier**

### CODE REVIEW

File: python/src/SymbolicFunction_doc.i.in [hunk 1]
```diff
@@ -55,7 +55,6 @@ Available functions:
 
     Available operators:
 
-    - = (assignment)
```
Comment (jschueller) [inline python/src/SymbolicFunction_doc.i.in]: In exprtk it's ":=", do we want to keep it ?

Comment (dbarbier) [reply python/src/SymbolicFunction_doc.i.in]: It is different with muparser and exprtk, thus I propose to drop it.  Advanced users will have to read exprtk documentation anyway.

**Comment by regislebrun** [comment]
+1

File: python/src/SymbolicFunction_doc.i.in (hunk 1 above)
Comment (regislebrun) [reply python/src/SymbolicFunction_doc.i.in]: ok

**Pull request merged**